spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/http/RestTemplateTransportClientFactoryTest.java

```diff
@@ -26,31 +26,31 @@
  */
 class RestTemplateTransportClientFactoryTest {
```

Comment:
I know this class was already here, but as you're introducing changes, could you please rename it to `RestTemplateTransportClientFactoryTests`?